app/components/emoji_picker/emoji_picker.js

```diff
@@ -161,23 +165,22 @@ export default class EmojiPicker extends PureComponent {
     };
 
     changeSearchTerm = (text) => {
-        const nextState = {
-            searchTerm: text,
-        };
```

Comment:
I think the `searchTerm` should still be `text` so that the text being entered by a user will not change unexpectedly in the textbox.